src/Exporter.php

```diff
@@ -120,7 +120,7 @@ public function shortenedExport($value)
             return sprintf(
                 '%s Object (%s)',
                 get_class($value),
-                count($this->toArray($value)) > 0 ? '...' : ''
+                method_exists($value, '__toString') ? (string)$value : (count($this->toArray($value)) > 0 ? '...' : '')
```

Comment:
I have to agree with @theseer, @GrahamCampbell. This is the same behavior that got @ircmaxell upset in https://github.com/sebastianbergmann/phpunit/pull/1793#issuecomment-121775187